src/app/components/publiclists/challengelist/challengelist.component.scss

```diff
@@ -1,34 +1,190 @@
-@import './variables.scss';
-@import './mixins.scss';
-
-.challengelist-container { 
-	padding:30px;
-  overflow:auto;
-  .challengelist-filters {
-  	padding:0px 0px 30px 0px;
-  	a {
-  		width:220px;
-  		display:inline-block;
-  		font-size:$fs-16;
-  		font-weight:$fw-light;
-  	}
+@import "styles/variables";
+
+.challenge-title-container {
+  padding-bottom: 0px;
+  ul.inline-list {
+    margin: 0px;
+    li {
+      margin-right: 40px;
+      min-height: 40px;
+    }
   }
-  .see-more {
-    text-align: center;
-    margin-top:20px;
-    font-size:$fs-16;
-    font-weight:$fw-light;
+}
+
+a.active-challenge {
+  color: $dark-black;
+  display: block;
+  height: 40px;
+  box-sizing: border-box;
+  border-bottom: 2px solid $highlight;
+}
+
+.ev-challenge-view {
+  margin-top: 0px;
+  padding-top: 30px;
+  margin-bottom: 20px;
+}
+
+.challenge-container {
+  margin: 0px 10px -30px 10px;
+}
+
+.eval-card {
+  min-height: 160px;
+}
+
+.phase-card {
+  padding-bottom: 1px;
+}
+
+.ev-dark-card {
+  background-color: $light-gray;
+}
+
+.challenge-page-title {
+  padding-bottom: 10px;
+}
+
+.ev-challenge-card {
+  min-height: 415px;
+}
+
+.ev-dashboard-card {
+  min-height: 190px;
+}
+
+.bg-img {
+  /* height: 120%; */
+  width: 100%;
+  margin: auto;
+  /* width: 109%; */
+  margin-top: -5%;
```

Comment:
Define margin like this instead `  margin: -5% auto auto;` (remove the above one.)

Reply:
done